src/inference_profiler.cc

```diff
@@ -618,12 +618,12 @@ InferenceProfiler::Profile(
   auto* custom_manager =
       dynamic_cast<CustomRequestScheduleManager*>(manager_.get());
   if (custom_manager) {
-    RETURN_IF_ERROR(custom_manager->InitCustomSchedule(request_count));
+    custom_manager->Start();
   } else {
     auto* request_rate_manager =
         dynamic_cast<RequestRateManager*>(manager_.get());
-    RETURN_IF_ERROR(
-        request_rate_manager->PerformWarmup(request_rate, request_count));
+    RETURN_IF_ERROR(request_rate_manager->PerformWarmup(
+        request_rate, warmup_request_count));
```

Comment:
This was actually an unrelated bug.

Previously, request rate mode was accidentally using the request count instead of the warmup request count for the number of warmup requests to send.